processing/src/test/java/io/druid/segment/filter/InFilterTest.java

```diff
@@ -138,7 +138,7 @@ public void testMissingColumn()
 
   private DimFilter toInFilter(String dim, String value, String... values)
   {
-    return new InDimFilter(dim, Lists.asList(value, values));
+    return new InDimFilter(dim, Lists.asList(value, values), null);
```

Comment:
while you're adding an extractionFn test to InFilterTest, could you please also change this to use the newer filter test style with `assertFilterMatches`? That'll make sure that filtered aggregators work too.